src/components/CustomButton/CustomButton.js

```diff
@@ -1,20 +1,34 @@
-import React, { Fragment } from "react";
-import ButtonStyle from "./CustomButtonStyle";
+import React from "react";
+import { createMuiTheme,  makeStyles, ThemeProvider } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import { lightBlue,  lightGreen } from '@material-ui/core/colors';
 import { FormattedMessage } from "react-intl";
 
+const useStyles = makeStyles((theme) => ({
+  margin: {
+    margin: theme.spacing(1),
+  },
+}));
+
+const theme = createMuiTheme({
+  palette: {
+    primary: lightGreen,
+    secondary: lightBlue
+  },
+});
+
 export default (props) => {
   const { modifier = "primary", titleId = "", title = "", ...rest } = props;
 
+  const classes = useStyles();
+
   return (
-    <Fragment>
-      <ButtonStyle />
-      <div className={`button ${modifier}`}>
-        <div className="button-wrapper">
-          <button {...rest}>
-            {titleId == "" ? title : <FormattedMessage id={titleId} />}
-          </button>
-        </div>
-      </div>
-    </Fragment>
+    <div>
```

Comment:
Maybe don't wrap it in`div`? Or did you mean for it to be block-level always?

Reply:
sure done